elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -3,36 +3,81 @@
+                name="{{name}}" value="{{key}}" {{^editable}}disabled{{/editable}}
+                {{#checked}}checked{{/checked}} id="{{name}}-{{key}}">
+
+            <label class="form-check-label d-flex align-items-center" for="{{name}}-{{key}}">
+                {{^hide_letter_keys}}<div class="pl-multiple-choice-key-label">({{key}})</div>{{/hide_letter_keys}}
+                <div class="ml-1 mr-1">{{{html}}}</div>
+            </label>
+
+            {{#display_feedback}}
+            <div class="pl-multiple-choice-feedback">
+                {{{feedback}}}
+            </div>
+            {{/display_feedback}}
+
+            {{#display_score_badge}}
+                {{#correct}}
+                    <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
+                {{/correct}}
+                {{#incorrect}}
+                    <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
+                {{/incorrect}}
+                {{#partial}}
+                    <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i></span>
+                {{/partial}}
+            {{/display_score_badge}}
         </div>
-        {{/display_feedback}}
-
-        {{#display_score_badge}}
-            {{#correct}}
-                <span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>
-            {{/correct}}
-            {{#incorrect}}
-                <span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>
-            {{/incorrect}}
-            {{#partial}}
-                <span class="badge badge-warning"><i class="far fa-circle" aria-hidden="true"></i></span>
-            {{/partial}}
-        {{/display_score_badge}}
+    {{/answers}}
+{{/radio}}
+{{^radio}}
+<script>
+    $(function(){
+        $(document).ready(function(){
+            $(".pl-dropdown-choice-{{uuid}}.active").each(function(){
+                $("#pl-multiple-choice-dropdown-button-{{uuid}}").text($(this).text());
+            });
+        });
+
+        $(".pl-dropdown-choice-{{uuid}}").click(function(){
+            // First, set dropdown button to display same text as selected button
+            $("#pl-multiple-choice-dropdown-button-{{uuid}}").text($(this).text());
+            
+            // Next, change active choice to be what the user just selected
+            $(".pl-dropdown-choice-{{uuid}}").each(function(){
+                $(this).removeClass("active");
+            });
+            $(this).addClass("active");
+
+            // Finally, set input value
+            $("#pl-dropdown-input-{{uuid}}").val($(this).attr("value"));
+        });
+    });
+</script>
+<div class="dropdown py-1 d-inline-flex">
+    <button class="btn btn-secondary dropdown-toggle" type="button" id="pl-multiple-choice-dropdown-button-{{uuid}}" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
@nwalters512 do you know if there's a way to force this button to have the same width as the corresponding dropdown menu? I tried setting the widths to be equal using some jQuery but it wound up looking weird. Not sure if this is something weird related to the way this works in bootstrap.

Reply:
@jonatanschroeder this is what I was having issues with. The width of the dropdown keeps changing based on the text inside (instead of remaining a constant width like the HTML `select` element does).

Reply:
I was checking out some options and indeed it doesn't seem like there is an easy option here. That said, I think if you provide some `min-width` (maybe use `10rem`, which is [the default for bootstrap menu items](https://github.com/twbs/bootstrap/blob/v4.6.2/dist/css/bootstrap.css#L3241)) this may not be a big deal.

Reply:
Check this option instead: https://github.com/snapappointments/bootstrap-select/

Reply:
@jonatanschroeder I started trying to integrate the bootstrap select in this commit: https://github.com/PrairieLearn/PrairieLearn/pull/6775/commits/145ec1bda04e2bc32f76ebd8cfcfe3ef3dd6400a

I thought it seemed really nice, but was having an issue where the LaTeX wouldn't render for individual options. That's really my only reservation to using this, really would be nice not to have to implement that functionality ourselves.

Reply:
Did you try disabling sanitizer?

Reply:
I'm playing around with a few options, I'll push a new change if I find something useful. I'll also add the yarn package.

Reply:
Latex math issue resolved in d2fadff32.